### PR TITLE
fix implementation of percentile die

### PR DESCRIPTION
**Pull request by BillWagner**
From a LiveFyre comment. (See the comment on "What's new in C# 7")

The implementation of the percentile die died not match the description in the text. This PR fixes that.

### CODE REVIEW

File: samples/snippets/csharp/new-in-7/patternmatch.cs [hunk 1]
```diff
@@ -9,13 +9,13 @@ namespace new_in_7
     #region 18_PercentileDie
     public struct PercentileDie
```
Comment (mairaw) [inline samples/snippets/csharp/new-in-7/patternmatch.cs]: Die or Dice? This distinction got me confused throughout the topic now that I went to see the comment.

File: samples/snippets/csharp/new-in-7/patternmatch.cs [hunk 2]
```diff
@@ -110,7 +110,7 @@ public static int DiceSum5(IEnumerable<object> values)
                         sum += val;
                         break;
                     case PercentileDie die:
-                        sum += die.Multiplier * die.Value;
+                        sum += die.TensDigit * 10 + die.OnesDigit;
```
Comment (mairaw) [inline samples/snippets/csharp/new-in-7/patternmatch.cs]: given that you're saying that one of the dice are labeled 00, 10, 20, 90, etc., you shouldn't have to multiply it here. Or you need to change the text to make both dice be only digits, no?

File: samples/snippets/csharp/new-in-7/patternmatch.cs (hunk 1 above)
Comment (BillWagner) [reply samples/snippets/csharp/new-in-7/patternmatch.cs]: I updated the markdown to explain dice as the plural of die, and changed the code.

File: samples/snippets/csharp/new-in-7/patternmatch.cs (hunk 2 above)
Comment (BillWagner) [reply samples/snippets/csharp/new-in-7/patternmatch.cs]: I updated it and clarified the markdown.

**Review by mairaw — APPROVED** [review]
LGTM

File: samples/snippets/csharp/new-in-7/patternmatch.cs [hunk 3]
```diff
@@ -6,16 +6,16 @@
...
+    #region 18_PercentileDice
+    public struct PercentileDice
     {
-        public int Value { get; }
-        public int Multiplier { get; }
+        public int OnesDigit { get; }
+        public int TensDigit { get; }
 
-        public PercentileDie(int multiplier, int value)
+        public PercentileDie(int tensDigit, int onesDigit)
```
Comment (mairaw) [inline samples/snippets/csharp/new-in-7/patternmatch.cs]: sorry, just notice this one still says Die

Comment (BillWagner) [reply samples/snippets/csharp/new-in-7/patternmatch.cs]: Doh. I must have committed before I did the last build and run test.

It's fixed now.

**Comment by BillWagner** [comment]
build failed with no error message. Closing and re-opening to trigger a new build.

**Pull request merged**